Binary files linux-2.6.19.2.orig/arch/cris/arch-v10/drivers/.axisflashmap.c.swp and linux-2.6.19.2/arch/cris/arch-v10/drivers/.axisflashmap.c.swp differ
diff -urN linux-2.6.19.2.orig/arch/cris/arch-v10/drivers/axisflashmap.c linux-2.6.19.2/arch/cris/arch-v10/drivers/axisflashmap.c
--- linux-2.6.19.2.orig/arch/cris/arch-v10/drivers/axisflashmap.c	2007-05-28 01:40:09.000000000 +0200
+++ linux-2.6.19.2/arch/cris/arch-v10/drivers/axisflashmap.c	2007-05-28 01:41:29.000000000 +0200
@@ -256,7 +256,7 @@
 
 /* If no partition-table was found, we use this default-set. */
 #define MAX_PARTITIONS         7  
-#define NUM_DEFAULT_PARTITIONS 3 
+#define NUM_DEFAULT_PARTITIONS 2 
 
 /*
  * Default flash size is 2MB. CONFIG_ETRAX_PTABLE_SECTOR is most likely the
@@ -270,15 +270,10 @@
 		.offset = 0
 	},
 	{
-		.name = "filesystem",
-		.size = 0x200000, 
+		.name = "rootfs",
+		.size = 0x600000, 
 		.offset = 0x200000
 	},
-	{
-		.name = "filesystem2",
-		.size = 0x400000, 
-		.offset = 0x400000
-	}
 };
 
 /* Initialize the ones normally used. */
Binary files linux-2.6.19.2.orig/arch/cris/arch-v10/drivers/axisflashmap.o and linux-2.6.19.2/arch/cris/arch-v10/drivers/axisflashmap.o differ
Binary files linux-2.6.19.2.orig/arch/cris/arch-v10/drivers/built-in.o and linux-2.6.19.2/arch/cris/arch-v10/drivers/built-in.o differ
